tests/e2e/upgrade/upgrade.wdio-spec.js

```diff
@@ -145,6 +149,28 @@ describe('Performing an upgrade', () => {
     const expected = TAG || `${utils.escapeBranchName(BRANCH)} (`;
     expect(await aboutPage.getVersion()).to.include(expected);
     await commonPage.logout();
+
+    // https://github.com/medic/cht-core/issues/9117
+    // install 'master' branch to make sure a new version can be installed from the build version
+
+    await loginPage.cookieLogin({
+      username: constants.USERNAME,
+      password: constants.PASSWORD,
+      createUser: false
+    });
+
+    await upgradeVersion('master');
+
+    expect(await upgradePage.getBuild()).to.include('alpha');
+    await commonPage.goToAboutPage();
+    await commonPage.waitForPageLoaded();
+    await (await aboutPage.aboutCard()).waitForDisplayed();
+    expect(await aboutPage.getVersion()).to.include('master');
+
+    // installing again the BRANCH version for further validations
+    await upgradeVersion(BRANCH);
```

Comment:
@garethbowen I had to upgrade the version to the build version after I installed `master` because if I didn't do it the next tests would fail. See [this](https://github.com/medic/cht-core/actions/runs/9684357961/job/26723377655?pr=9215) failed job

Do you think that it will be better just to re-install the build version over the top of itself instead of installing `master`? 
The problem with doing that is that we can not verify that the second time that we installed it was successfully installed it

Reply:
Yeah great point. Whatever we do is really risky because if the downgrade fails for any reason then the rest of the tests will fail in a confusing way, and we don't really need to test the downgrade option because we don't support that in production!

You could either run this test at the end of the suite so it doesn't matter what state it's left in, OR you could run this test in its own suite. I think I prefer the second option because upgrade testing is so challenging to get right, you really want to know you're starting with a clean slate.

What do you think?

Reply:
Yes, I am agree
I will isolate it and also add it at the end, just in case 😅 
Thank you!